tensorflow/core/kernels/segment_reduction_ops_gpu.cu.h

```diff
@@ -1029,6 +1050,362 @@ struct SparseSegmentGradFunctor<GPUDevice, T, Index, SegmentId> {
   }
 };
 
+template <typename Tindices_internal>
```

Comment:
UpperCamelCase, even for `Internal`, though I think we should use a more meaningful name.  For this function on its own, `Tindices` alone is fine.

Reply:
I've renamed all `Tindices_internal` to `TindicesCompact` (the best I could come up with).